plugins/source/unit_test_validators.go

```diff
@@ -0,0 +1,99 @@
+package source
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/cloudquery/plugin-sdk/schema"
+	pluralize "github.com/gertd/go-pluralize"
+)
+
+type Validator func(t *testing.T, plugin *Plugin, resources []*schema.Resource) error
```

Comment:
I think it should either have `t` or `error`, not both? You can check if it errored by calling `t.Failed()`